### DISCUSSION TITLE
Is there a way to host multiple documentation sets?

### DISCUSSION THREAD

**Discussion by donvito** [fuma-nama/fumadocs#594] [Q&A]
I am thinking of using fumadocs to create short mini courses. Is there a way to use it for multiple documentation sets? My ideas is for each course it will have its own content/docs folder then from the home page, it will contain the list of available courses. 

<img width="1115" alt="Screenshot 2024-07-12 at 5 00 21 PM" src="https://github.com/user-attachments/assets/80623896-f4a1-481a-9b40-f37afabf09ac">

**Comment by fuma-nama** [comment]
You can use multiple page trees with Root Toggle (https://fumadocs.vercel.app/docs/headless/page-conventions#multiple-page-trees), or create multiple content types and make a separate layout for each of them: https://fumadocs.vercel.app/docs/mdx/multiple-types

**Reply by donvito** [reply]
It seems the code is different now

```
export const { getPage, getPages, pageTree } = loader({
  baseUrl: '/docs',
  rootDir: 'docs',
  source: createMDXSource(map),
});
```

I tried adding, but it didn't work.

```
export const docs2 = loader({
  baseUrl: '/docs2',
  rootDir: 'docs2',
  source: createMDXSource(map),
});
```

**Reply by fuma-nama** [reply]
You still have to create another layout for your docs2 content type.
Considering your use case, multiple page tree would be a better choice. You can mark them as a root folder and configure the root toggle component to allow navigating between them
